script.js

```diff
@@ -1,11 +1,24 @@
 // Create an array called `colors` that contains three colors.
+var colors = ["red", "blue", "grey"]
 
 // Log the last item in the array to the console.
+console.log(colors);
 
 // Create a `for` loop that prints out "My favorite color is [insert color here]" for each item in `colors`.
+for ( var i = 0; i < colors.length; i++) {
+  console.log("My favorite color is " + colors)
+};
+
 
```

Comment:
very close. should be `colors[i])` in your `console.log()`. Be sure you're testing your code.